README.md

```diff
@@ -7,6 +7,10 @@ A module for interacting with the discordbots.org API
 ## Documentation
 Documentation can be found [here](https://discordbots.org/api/docs#jslib)
 
+### Dependencies 
```

Comment:
The readme doesn't need a dependencies section.
npm handles those via package.json